doc/ant-xgboost_user_guide.md

```diff
@@ -0,0 +1,164 @@
+# _user guide:_ Ant-XGBoost on sqlflow
+
+## Overview
+
+[Ant-XGBoost](https://github.com/alipay/ant-xgboost) is fork of [dmlc/xgboost](https://github.com/dmlc/xgboost), which is maintained by active contributors of dmlc/xgboost in Alipay Inc.
+
+Ant-XGBoost extends `dmlc/xgboost` with the capability of running on Kubernetes and automatic hyper-parameter estimation. 
+In particular, Ant-XGBoost includes `auto_train` methods for automatic training and introduces an additional parameter `convergence_criteria` for generalized early stopping strategy.
+
+## Tutorial
+We provide an [interactive tutorial](../example/jupyter/tutorial_antxgb.ipynb) via jupyter notebook, which can be run out-of-the-box in [sqlflow playground](https://play.sqlflow.org).
+If you want to run it locally, you need to install sqlflow first. You can learn how to install sqlflow at [here](../doc/installation.md).
+
+## Concepts
+### Estimators
+We provide various XGBoost estimators for better user experience.
+All of them are case-insensitive and sharing same prefix `xgboost`. They are listed below.
+
+* xgboost.Estimator
+
+  General estimator, with this `train.objective` should be defined explicitly.
+  
+* xgboost.Classifier 
+    
+  Estimator for classification task, works with `train.num_class`. Default value is binary classification.
+  
+* xgboost.BinaryClassifier 
+    
+  Estimator for binary classification task, set `train.objective` to `binary:logistic`.
+
+* xgboost.MultiClassifier 
+    
+  Estimator for multi classification task, set `train.objective` to `multi:softprob`, should work with `train.num_class` > 2.
+
+* xgboost.Regressor 
+    
+  Estimator for regression task, set `train.objective` to `reg:squarederror`(`reg:linear`). 
+
+<br>
+
+### Columns
+
+#### Feature Columns
+For now, two feature column schemas are available.
+
+First one is `dense schema`, which concatenate numeric table columns transparently, such as `COLUMN f1, f2, f3, f4`.
+
+Second one is `sparse key-value schema`, which received string sparse feature formatted like `$k1:$v1,$k2:$v2,...`.
+This schema is decorated with keyword `SPARSE`, such as `COLUMN SPARSE(col1)`.
+
+#### Label Column
+Following general sqlflow syntax, label clause of AntXGBoost is formatted in  `LABEL $label_col`. 
+
+#### Group Column
+In training mode, group column can be declared in a separate column clause. Group column is identified by keyword `group`, such as `COLUMN ${group_col} FOR group`.
+
+#### Weight Column
+As group column schema, weight column is identified by keyword `weight`, such as `COLUMN ${weight_col} FOR weight`.
+
+#### Result Columns
+Schema of straightforward result (class_id for classification task, score for regression task) is following general sqlflow syntax(`PREDICT ${output_table}.${result_column}`).
+
+In addition, we also provide supplementary information of XGBoost prediction. They can be configured with `pred.attributes`.
+
+##### append columns
```

Comment:
The title is too deep, do you want to use `*` to introuce how to use `append_columns` ?

Reply:
Done